project/plugins.sbt

```diff
@@ -15,7 +15,7 @@ addSbtPlugin("org.foundweekends" % "sbt-bintray" % "0.5.4")
 // scripted testing
 libraryDependencies += "org.scala-sbt" %% "scripted-plugin" % sbtVersion.value
 
-libraryDependencies += "com.thesamet.scalapb" %% "compilerplugin" % "0.8.4"
+libraryDependencies += "com.thesamet.scalapb" %% "compilerplugin" % "0.9.0-M1"
```

Comment:
Let's not merge before 0.9.0 is properly released

Reply:
Can we instead merge and release a milestone version? Or maybe have a snapshot release. That way, we can also work on Scala 2.13 in play-grpc.

Reply:
@raboof what about enabling publishing from pull requests?  that way we can consume this change and iterate in a PR in play-grpc, and then land both PRs when scalapb has a stable release.

Reply:
I don't think we should publish every commit to every pull request, but indeed publishing snapshots is fine - I `sbt +publish`ed this branch, introducing 0.6.0+11-23237fc2 on bintray.

Reply:
OK, thanks.